cluster/ci/config/prow/jobs/README.md

```diff
@@ -0,0 +1,130 @@
+ci-operator templates
+=====================
+
+
+E2E tests
+---------
+
+The preferred way to write e2e tests is using `ci-operator`.  See the
+documentation for details on how to download, build, and execute it:
+
+https://github.com/openshift/ci-operator.git
+
+Many projects already have a `ci-operator` configuration file, which takes care
+of most of the CI process: downloading the source, building binaries, building
+RPMs, creating images, executing unit tests, etc.  These can be built upon for
+e2e test, requiring little or no modification.
+
+
+# Provisioning a cluster
+
+Contrary to other types of tests, E2E tests usually require a cluster, not just
+a single container.  While there aren't yet native primitives in `ci-operator`
+for cluster provisioning, it provides one open-ended feature that can be
+leveraged to accomplish that: template steps.
+
+Template steps allow the creation of arbitrary objects (e.g. pods) in the
+cluster where the CI pipeline is executed.  This directory already contains a
+few templates that can be used either directly or (rarely necessary, in
+practice) as a reference:
+
+TODO: describe these in detail: purpose, inputs, …
+
+- `cluster-launch-e2e.yaml`:
+- `cluster-launch-installer-e2e.yaml`:
+- `cluster-launch-src.yaml`:
+- `master-sidecar.yaml`:
+- `os.yaml`:
+
+
+# Using a template
+
+A template can be used with the `--template` argument.  The name of the
+template, or the name of the file without the extension, becomes the name of
+the stage, so it can be used as an argument to `--target`.  This does not
+require modifying the configuration file.
```

Comment:
This assumes the knowledge that CI operator uses a mandatory config file, so I would perhaps help a reader a little by mentioning CI operator again.

Reply:
Updated.